quantum/keymap_extras/keymap_swedish.h

```diff
@@ -76,7 +76,7 @@
 #define SE_APOS_MAC KC_NUBS  // '
 #define SE_AT_MAC   KC_BSLS  // @
 #define SE_BSLS_MAC ALGR(LSFT(KC_7)) // '\'
-#define SE_DLR_MAC  LSFT(KC_4) // $
+#define SE_DLR_MAC  ALGR(KC_4) // $
```

Comment:
Is this Mac specific?   Or does it apply to Windows, Mac and Linux? 

if it only applies to Mac, I'm not sure it's a good change, and we should probably use `keymap_swedish_macos.h` instead?

Reply:
It works on Windows, Linux and Mac OS